science/nds2-client-swig/Portfile

```
@@ -141,7 +161,7 @@ foreach v {27 35 36 37 38} {
   set python.branch           [string range ${python.version} 0 end-1].[string index ${python.version} end]
   set python.bin              ${prefix}/bin/python${python.branch}
   set python.prefix           ${frameworks_dir}/Python.framework/Versions/${python.branch}
-  set python.site_packages    "${python.prefix}/lib/python${python.branch}/site-packages"
+    set python.site_packages    "${python.prefix}/lib/python${python.branch}/site-packages"
```

You should revert this change. Although I would like to see the whitespace of this port fixed to conform to the modeline (i.e. 4 spaces per indentation) it should be done to the entire portfile at once without any functional changes, so let's leave that for another day.
```suggestion
  set python.site_packages    "${python.prefix}/lib/python${python.branch}/site-packages"
```

---

I have added this to my latest commit